sympy/geometry/tests/test_ellipse.py

```diff
@@ -9,6 +9,18 @@
 from sympy import integrate
 from sympy.functions.special.elliptic_integrals import elliptic_e
 
+def test_object_from_equation():
+    from sympy.abc import x, y
+    assert Circle.object_from_equation(x ** 2 + y ** 2 + 3 * x + 4 * y - 8) == Circle(Point2D(-3 / 2, -2), sqrt(57) / 2)
```

Comment:
`-3/2` is a float. You should use `-S(3)/2`.

Reply:
@sidhantnagpal  Done that now (you'll be able to see that in the next commit). I even tried to make the function return rationals in the form of `S..` but I was unable to do that. But I don't think that would be necessary or needed in any way. What do you think?